app.js

```diff
@@ -44,7 +46,7 @@ async function renderScenarios(data) {
   anchor.append(section);
   document
     .querySelectorAll(".game ul li")
-    .forEach((e, i) => (e.onclick = () => data.renderInputs(i)));
+    .forEach((e, i) => (e.onclick = () => data.renderInputs(scenarios, i)));
```

Comment:
You have access to the same exact information that `story.Scenario_title` is carried on, so at this point and time you actually already have to the logic in this function to tell the collectScenarios which story this one is...

I would have passed in ...data.renderInputs(story.Scenario, story.Scenario_title, i)

But your code works so let's send it!